x-pack/plugins/monitoring/server/index.ts

```diff
@@ -27,6 +27,7 @@ export const config: PluginConfigDescriptor<TypeOf<typeof configSchema>> = {
       container: true,
       ccs: {
         enabled: true,
+        remotePatterns: true,
```

Comment:
should we not expose the remote names to the browser?  If we don't want to the implication is that there would be a "*" search when the user is entering a filter query in a rule:

<img width="618" alt="Screen Shot 2022-04-07 at 5 00 37 PM" src="https://user-images.githubusercontent.com/1676003/162305077-50c361cf-668a-435f-8676-e89e88aec8b2.png">